server/zanata-frontend/src/app/editor/components/DashboardLink/DashboardLink.test.js

```diff
@@ -6,7 +6,7 @@ import DashboardLink from '.'
 describe('DashboardLinkTest', () => {
   it('DashboardLink markup', () => {
     const profilePicSrc =
-      'http://www.emoji-cheat-sheet.com/graphics/emojis/smiling_imp.png'
+      '//www.emoji-cheat-sheet.com/graphics/emojis/smiling_imp.png'
```

Comment:
I know this is just test code, but it seems the URL is no longer pointing at an image (especially when you put `https` in front of it). This works now: https://www.webpagefx.com/tools/emoji-cheat-sheet/graphics/emojis/smiling_imp.png

Reply:
Ok thanks, will update the code.